serverlessworkflow/modules/ROOT/pages/eventing/event-correlation-with-workflows.adoc

```diff
@@ -1 +1,205 @@
-//= Event correlation with workflows
\ No newline at end of file
+= Event correlation with workflows
+=====================================
+:compat-mode!:
+// Metadata:
+:description: Event Correlation in Serverless Workflow
+:keywords: kogito, workflow, serverless, correlation, association, domain, events
+
+[[ref-example-jq-expression-switch-conditions]]
+
+Event correlation plays a big role in large event-driven applications, it allows to match one or more events with a particular workflow instance. Correlation  rules can be used when defining consumed events as a way to match a workflow instance, it is a more practical alternative to the internal identifier `processInstanceId`, since any external domain identifier can be used as a correlation.
```

Comment:
```suggestion
Event correlation plays a significant role in large event-driven applications. It allows matching one or more events with a particular workflow instance. Correlation rules with domain information can be used when defining consumed events to match a workflow instance. It is a more practical alternative to the internal identifier `processInstanceId` since you can use any external domain identifier as a correlation.
```

Reply:
+1